sample/Kmd.Logic.CitizenDocuments.Client.Sample/Program.cs

```diff
@@ -74,22 +75,34 @@ private static async Task<string> Run(AppConfiguration configuration)
                 var citizenDocumentClient = new CitizenDocumentsClient(httpClient, tokenProviderFactory, configuration.Citizen);
                 var uploadDocument = await citizenDocumentClient.UploadAttachmentWithHttpMessagesAsync(configuration.ConfigurationId, configuration.RetentionPeriodInDays, configuration.Cpr, configuration.DocumentType, configuration.Document, configuration.DocumentName).ConfigureAwait(false);
 
-                Log.Information("The {DocumentType} document with id {DocumentId} and file access page url {FileAccessPageUrl} is uploaded successfully", uploadDocument.DocumentType, uploadDocument.DocumentId, uploadDocument.FileAccessPageUrl);
+                Stream stream = File.OpenRead(configuration.DocumentName);
+
+                var uploadWithLargeSizeDocument = await citizenDocumentClient.UploadFileAttachmentDirectlyToStorageAsync(stream, new CitizenDocumentUploadRequestModel
+                {
+                    SubscriptionId = new Guid(configuration.SubscriptionId),
+                    CitizenDocumentConfigId = new Guid(configuration.ConfigurationId),
+                    Cpr = configuration.Cpr,
+                    DocumentType = configuration.DocumentType,
+                    RetentionPeriodInDays = configuration.RetentionPeriodInDays,
+                    DocumentName = configuration.DocumentName,
+                }).ConfigureAwait(false);
+
+                Log.Information("The {DocumentType} document with id {DocumentId} and file access page url {FileAccessPageUrl} is uploaded successfully", uploadWithLargeSizeDocument.DocumentType, uploadWithLargeSizeDocument.DocumentId, uploadWithLargeSizeDocument.FileAccessPageUrl);
 
                 var sendDocument = await citizenDocumentClient.SendDocumentWithHttpMessagesAsync(new SendCitizenDocumentRequest
                 {
                     ConfigurationId = new Guid(configuration.ConfigurationId),
                     SendingSystem = configuration.SendingSystem,
                     Cpr = configuration.Cpr,
                     DocumentType = configuration.SendDocumentType,
-                    CitizenDocumentId = uploadDocument.DocumentId,
+                    CitizenDocumentId = uploadWithLargeSizeDocument.DocumentId,
                     Title = configuration.Title,
-                    DigitalPostCoverLetterId = uploadDocument.DocumentId,
-                    SnailMailCoverLetterId = uploadDocument.DocumentId,
+                    DigitalPostCoverLetterId = uploadWithLargeSizeDocument.DocumentId,
+                    SnailMailCoverLetterId = uploadWithLargeSizeDocument.DocumentId,
                 }).ConfigureAwait(false);
 
                 Log.Information("The document is sent successfully and doc2mail provider response message id is {MessageId}", sendDocument.MessageId);
-
+                stream.Dispose();
```

Comment:
I think this one should be done in more safe manner. Like wrapping stream in `using`

Reply:
Removed stream.Dispose